db/db_test2.cc

```diff
@@ -468,6 +468,8 @@ namespace {
     }
 
     // Ensure that given keys don't exist
+    values.clear();
+    values.resize(keys_must_not_exist.size());
```

Comment:
How is that resize works here on PinnableSlice?

Reply:
this is a vector of std::string so you can freely resize it